tests/assets/network-tab/network.html

```diff
@@ -12,26 +12,32 @@
       }
     </style>
     <script src='script.js'></script>
-    <script>fetch('/api/endpoint')</script>
-    <script>fetch('/call-with-query-params?param1=value1&param1=value2&param2=value2')</script>
     <script>
       const body = JSON.stringify({
         data: {
           key: 'value',
           array: ['value-1', 'value-2'],
         },
       });
+      const fetches = [
+        fetch('/api/endpoint'),
+        fetch('/call-with-query-params?param1=value1&param1=value2&param2=value2'),
+        fetch('/post-data-1', {
+          method: 'POST',
+          headers: { 'content-type': 'application/json' },
+          body,
+        }),
+        fetch('/post-data-2', {
+          method: 'POST',
+          headers: { 'Content-Type': 'application/json' },
+          body,
+        }),
+      ];
+      window.donePromise = Promise.all(fetches.map(f => f.then(r => r.body()).catch(e => {})));
```

Comment:
this means we serialise the bodies back to the client - maybe we should return just null instead?